impl/src/main/java/org/jboss/weld/logging/BeanLogger.java

```diff
@@ -522,4 +522,8 @@
     @Message(id = 1570, value = "Invalid BeanConfigurator setup - no callback was specified for {0}", format = Format.MESSAGE_FORMAT)
     IllegalStateException noCallbackSpecifiedForCustomBean(Object bean);
 
+    @LogMessage(level = Level.WARN)
+    @Message(id = 1571, value = "Proxy for {0} created in {1} because {2}.", format = Format.MESSAGE_FORMAT)
```

Comment:
Just one thought here - user doesn't know/care where we store the proxies so it might not give him enough info.
E.g. they don't know we create proxies in the same package as was the original class.
Perhaps this msg would be more clear?
`Proxy for {0} created in package {1} which is different from the original one because {2}.`